staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2555,7 +2555,7 @@ type ContainerStateTerminated struct {
 	// Time at which the container last terminated
 	// +optional
 	FinishedAt metav1.Time `json:"finishedAt,omitempty" protobuf:"bytes,6,opt,name=finishedAt"`
-	// Container's ID in the format 'docker://<container_id>'
+	// Container's ID in the format '<container_id>'
```

Comment:
based on https://github.com/kubernetes/kubernetes/blob/d4b49be81a28a2de1c63bbd84ceb1bf3660fa983/pkg/kubelet/container/runtime.go#L214-L223, it looks like this is supposed to have the structure `<type>://<container_id>`

```suggestion
	// Container's ID in the format '<type>://<container_id>'
```